shared/src/housing.ts

```diff
@@ -7,8 +9,8 @@ export type HousingPropertyInterestDetails = {
   stateCode: number;
 };
 
-export type HousingMunicipality = {
-  name: string;
-  id: string;
-  county: string;
+export type CommunityAffairsAddress = {
```

Comment:
The previous municipality type was not being used, so it was deleted. The diff makes it looks like a drop-in replacement but `CommunityAffairsAddress` is actually a new type